app/src/main/java/com/example/android/moviedbtrainingapp/view/MainActivity.java

```diff
@@ -1,213 +0,0 @@
-package com.example.android.moviedbtrainingapp.view;
-
-import android.content.Intent;
-import android.os.Bundle;
-import android.support.v4.widget.SwipeRefreshLayout;
-import android.support.v7.app.AppCompatActivity;
-import android.support.v7.widget.LinearLayoutManager;
-import android.support.v7.widget.RecyclerView;
-import android.view.Menu;
-import android.view.MenuItem;
-import android.view.View;
-import android.widget.ProgressBar;
-import android.widget.Toast;
-
-import com.example.android.moviedbtrainingapp.R;
-import com.example.android.moviedbtrainingapp.model.general.Movie;
-import com.example.android.moviedbtrainingapp.model.general.MoviesResponse;
-import com.example.android.moviedbtrainingapp.model.utils.Constants;
-import com.example.android.moviedbtrainingapp.presenter.MovieRestManager;
-import com.example.android.moviedbtrainingapp.presenter.adapter.MovieAdapter;
-import com.example.android.moviedbtrainingapp.presenter.callback.MovieCallback;
-import com.facebook.stetho.Stetho;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import butterknife.BindView;
-import butterknife.ButterKnife;
-import rx.Observable;
-import rx.Observer;
-import rx.android.schedulers.AndroidSchedulers;
-import rx.schedulers.Schedulers;
-
-public class MainActivity extends AppCompatActivity implements MovieAdapter.MovieClickListener {
```

Comment:
Instead of implementing MovieClickListener here you should create interface for view with method like 'onMovieSelected' and it should wait for movie id as an argument.